GETTING-STARTED.md

```diff
@@ -1,166 +1,307 @@
-
-- Vive wands: Hold the application menu buttons on both controllers
-- Valve Index Controllers: Hold down down both Thumbsticks
-
-## 4. Menus
-
-When you start the game up, you'll probably want to navigate the game menus. Below the Edit Mode lock button is a Menu button you can press to enter menu mode.
-
-In menu mode all the other controls are disabled and your controller's buttons are just used to navigate the menu:
-
-- Vive wands
-  - Sliding your finger along the trackpad will navigate through menu items.
-  - Pressing on the edge of the trackpad will navigate through menu items.
-  - Pressing the center of the trackpad will select a menu item.
-  - Pressing the application menu button will go back/go up/exit menus.
-- Valve Index Controllers
-  - Moving the joystick will navigate through menu items.
-  - Sliding your finger along the trackpad will navigate through menu items.
-  - Pressing the A button will select a menu item.
-  - Pressing the B button will go back/go up/exit menus.
-  - Pressing the joystick will toggle nested menus open/closed.
-- Oculus Touch
-  - Moving the joystick will navigate through menu items.
-  - Pressing the A/X button will select a menu item.
-  - Pressing the B/Y button will go back/go up/exit menus.
-  - Pressing the joystick will toggle nested menus open/closed.
-- WMR
-  - Moving the joystick will navigate through menu items.
-  - Sliding your finger along the trackpad will navigate through menu items.
-  - Pressing the application menu button will go back/go up/exit menus.
-  - Pressing the joystick will toggle nested menus open/closed.
-
-Be sure to turn off menu mode when you leave the menu and want to control your ship. Menu mode is exclusively for controlling the main menu, pause menu, and in-game help screen.
-
-## 5. Axis bindings
-
-In order for the throttle, joystick, 6dof controller, and map to work you will need to bind their axis in the controls
-
-![Controls](Images/ScreenshotControls.png)
-
-Unlock edit mode while in menu mode to display the cockpit panel. This panel has a set of buttons to send virtual throttle/joystick axis inputs and joystick button inputs to the game. This will allow you to bind controls for the joystick and throttle even when they are not present. The Map and Misc tabs can be used to bind additions bindings for the galaxy map and other controls.
-
-![Controls Panel](Images/ScreenshotControlsPanel.png)
-
-Make sure you have the following bound. Only pitch should be inverted, all others should be set to regular.
+## 3. Launch Programs (in order)
 
-- `[JOY RZ-AXIS]` on yaw axis
-- `[JOY X-AXIS]` on roll axis
-- `[JOY Y-AXIS]` on pitch axis
-- `[JOY Z-AXIS]` on throttle axis
-- `[JOY RX-AXIS]` on the lateral thruster axis
-- `[JOY RY-AXIS]` on the vertical thruster axis
-- `[JOY U-AXIS]` on the forward/backward thruster axis
+0. If you have an Oculus, [do the workaround](OCULUS-WORKAROUND.md)
+1. Start Steam VR (usually done within your headset)
```

Comment:
Ah, did not know that! 

Since I referenced steam VR in step 1b, maybe this is unnecessary? Or perhaps it could be re-worded to say "make sure Steam VR is running" on your headset?

Reply:
I guess this would be good.

```suggestion
1. Turn on your VR hardware and launch Steam VR. If you don't have a native Steam VR device it can be launched separately from within your headset.
```